scripts/crawl.py

```diff
@@ -64,15 +64,20 @@ def parse_args():
 if __name__ == "__main__":
     github_token, config_path, verbose, force = parse_args()
     try:
-        if verbose:
-            print("==================== Zenodo Crawler Running ====================" + os.linesep)
-        ZenodoCrawler = ZenodoCrawler(github_token, config_path, verbose, force)
-        ZenodoCrawler.run()
+        # if verbose:
+        #     print("==================== Zenodo Crawler Running ====================" + os.linesep)
+        # ZenodoCrawler = ZenodoCrawler(github_token, config_path, verbose, force)
+        # ZenodoCrawler.run()
+        #
+        # if verbose:
+        #     print(os.linesep + "==================== OSF Crawler Running ====================" + os.linesep)
+        # OSFCrawler = OSFCrawler(github_token, config_path, verbose, force)
+        # OSFCrawler.run()
 
         if verbose:
-            print(os.linesep + "==================== OSF Crawler Running ====================" + os.linesep)
-        OSFCrawler = OSFCrawler(github_token, config_path, verbose, force)
-        OSFCrawler.run()
+            print(os.linesep + "==================== FRDR Crawler Running ====================" + os.linesep)
+        FRDRCrawler = FrdrCrawler(github_token, config_path, verbose, force)
+        FRDRCrawler.run()
```

Comment:
The three crawlers should be kept active, maybe the scripts could be updated in a separate PR.